### PR TITLE
(MODULES-8142) - Addition of support for SLES 15

### PR DESCRIPTION
**Pull request by david22swan**
net-telnet removed from gemfile as it does not need to be declared.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ group :development do
   gem "puppet-module-win-dev-r#{minor_version}",       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-lint-i18n",                              require: false
   gem "net-telnet", '0.1.0',                           require: false if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.3.0')
-  gem "net-telnet",                                    require: false if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')
+  gem "net-telnet", '0.1.1',                           require: false if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')
```
Comment (tphoney) [inline Gemfile]: also edit the .sync.yml or this will be lost. why the change ? does this clash with net-telnet in puppet agent ?

Comment (tphoney) [reply Gemfile]: .sync.yml still needs to be edited. Does this affect other modules ????

Comment (david22swan) [reply Gemfile]: Updated the sync file.
It conflicted with the specinfra gem as it pins net-telnet to 0.1.1

Comment (tphoney) [reply Gemfile]: Does this affect other modules ?

Comment (david22swan) [reply Gemfile]: Not that I've seen. inifile is the only one that seems to have net-telnet specifically set

**Comment by tphoney** [comment]
do you have the results from the ad-hoc pipetline?

**Comment by david22swan** [comment]
screen shot 2018-12-17 at 11 57 12 am
